connectors/sources/tests/fixtures/oracle/fixture.py

```diff
@@ -48,33 +51,42 @@ def inject_lines(table, cursor, start, lines):
 
 def load():
     """N tables of 10001 rows each. each row is ~ 1024*20 bytes"""
-    connection = cx_Oracle.connect(
-        user="system", password="oracle", dsn=DSN, encoding="UTF-8"
-    )
-    cursor = connection.cursor()
-    cursor.execute("CREATE USER admin IDENTIFIED by Password_123")
-    cursor.execute("GRANT CONNECT, RESOURCE, DBA TO admin")
-    connection.commit()
-
-    connection = cx_Oracle.connect(
-        user="admin", password="Password_123", dsn=DSN, encoding="UTF-8"
-    )
-    cursor = connection.cursor()
-    cursor.execute(
-        r"ALTER DATABASE DATAFILE '/u01/app/oracle/oradata/XE/system.dbf' AUTOEXTEND ON maxsize unlimited"
-    )
-    for table in range(NUM_TABLES):
-        print(f"Adding data in {table}...")
-        sql_query = f"CREATE TABLE customers_{table} (id int, name VARCHAR(255), age int, description long, PRIMARY KEY (id))"
-        cursor.execute(sql_query)
-        for i in range(10):
-            inject_lines(table, cursor, i * 1000, 1000)
-    connection.commit()
+    retry = 1
+    while retry <= RETRIES:
+        try:
+            connection = oracledb.connect(
+                user="system", password="Password_123", dsn=DSN, encoding="UTF-8"
+            )
+            cursor = connection.cursor()
+            cursor.execute("CREATE USER admin IDENTIFIED by Password_123")
+            cursor.execute("GRANT CONNECT, RESOURCE, DBA TO admin")
+            connection.commit()
+
+            connection = oracledb.connect(
+                user="admin", password="Password_123", dsn=DSN, encoding="UTF-8"
```

Comment:
`user`, `password` and `encoding` seem to be constant and are used at multiple places. You could create constants and reuse them.